packages/manager/apps/pci-instances/src/components/pciCard/PciCardShowcase.component.tsx

```diff
@@ -0,0 +1,236 @@
+import {
```

Comment:
Please delete this file as we don't want to merge boilerplate/template code :)
It could be used later in an UI library (storybook) but not in a µapp